test/pretty-printer/ppTest.ml

```diff
@@ -10,21 +10,21 @@ let generator =
            let lowercase_id =
              string_size ~gen:(char_range 'a' 'z') (return 1)
            in
+           let open Expr in
```

Comment:
Please don't do this. `Expr` and `Type` are designed to be used mostly in their fully qualified form.
```suggestion
```

Reply:
`let open Expr in` still needs a fix.